airmozilla/manage/templates/manage/survey_edit.html

```diff
@@ -9,19 +9,8 @@
   {% compress css %}
   <link href="{{ static('select2/select2.css') }}" rel="stylesheet" type="text/css">
   <link href="{{ static('select2/select2-bootstrap.css') }}" rel="stylesheet" type="text/css">
+  <link href="{{ static('manage/css/survey.css') }}" rel="stylesheet" type="text/css">
```

Comment:
I'm almost embarrassed by this but you need to do only `select2.css` and `select2-bootstrap.css` together in one block of `compress css`/`endcompress`.

The reason is that if you look for places where `select2-bootstrap.css` is used, throughout the project, you'll see that there are many places where it uses these two together. When the `compress css` thing works, it turns that into one file with a unique name. E.g. a3ed10efaa2.css and that gets reused throughout the management pages. So, if you make a combination of those two plus one more it gets a different unique name and the browser won't know it's already downloaded parts of it so you get excess downloaded. 

The solution would be to do this:
```html
{% compress css %}
<link href="{{ static('select2/select2.css') }}" rel="stylesheet" type="text/css">
<link href="{{ static('select2/select2-bootstrap.css') }}" rel="stylesheet" type="text/css">
{% endcompress %}
{% compress css %}
<link href="{{ static('manage/css/survey.css') }}" rel="stylesheet" type="text/css">
{% endcompress %}
```